metagraph/src/cli/stats.cpp

```diff
@@ -226,6 +247,70 @@ void print_stats(const Annotator &annotation) {
     if (const auto *brwt = dynamic_cast<const BRWT *>(mat))
         print_brwt_stats(*brwt);
 
+    if (config.print_counts_hist || config.count_quantiles.size()) {
+        std::cout << "===================== COUNTS STATS =====================" << std::endl;
+        if (!dynamic_cast<const annot::ColumnCompressed<> *>(&annotation)) {
+            logger->error("Printing statistics for counts is currently only supported for"
+                          " ColumnCompressed annotations");
+            exit(1);
+        }
+        // the annotation is not needed anymore -> free memory
+        anno_p.reset();
+
+        std::cout << fmt::format("Column-index\tLabel\tNum-counts");
```

Comment:
Maybe change the order to "Index NumCounts Label"? The label could be of any length, so it won't misalign the table columns if this happens.

Reply:
Though I guess it'll be misaligned anyway since "Column-index" is a long header name

Reply:
Yeah, it's tab-separated, so will be easily parsable anyway